src/Html/Column.php

```diff
@@ -20,6 +20,36 @@ public function __construct($attributes = [])
         $attributes['orderable']  = isset($attributes['orderable']) ? $attributes['orderable'] : true;
         $attributes['searchable'] = isset($attributes['searchable']) ? $attributes['searchable'] : true;
 
+        // Allow methods override attribute value
+        foreach($attributes as $attribute => $value) {
+            $method = 'parse' . ucfirst(strtolower($attribute));
+            if(method_exists($this, $method)) {
+                $attributes[$attribute] = $this->$method($value);
+            }
+        }
+
         parent::__construct($attributes);
     }
+
+    /**
+     * Parse Render
+     *
+     * @param $value
+     *
+     * @return string
+     */
+    public function parseRender($value)
+    {
+        $value = $value ?: $this->config->get('datatables.render_template', 'datatables::action');
+
```

Comment:
`$this->config->get()`.
Tests will fail because we don't have config here. Maybe we can use `app('view')` and `app('config')` and just mock the app function on tests.